### PR TITLE
News page

### PR DESCRIPTION
**Pull request by isaacabraham**

### CODE REVIEW

File: docs/faq-server.md [hunk 1]
```diff
@@ -3,28 +3,24 @@
...
 
- This is also shown in the table below. Both Giraffe's HTTP Handler and  and Suave's WebPart are very similar abstractions; this allows a simple migration path between either server. Meanwhile Saturn, which lives "on top" of Giraffe, provides a set of new higher-level abstractions which delegate down to Giraffe's HTTP Handler.
+Saturn, which lives "on top" of Giraffe, provides a set of new higher-level abstractions which delegate down to Giraffe's HTTP Handler.
 
-| | Saturn | Giraffe | Suave |
-|-|:-:|:-:|:-:|
-| ASP .NET compatible? | Yes | Yes | No |
-| F#-first? | Yes | Yes | Yes |
-| Core Abstractions | scope, controller and HttpHandler | HttpHandler | WebPart
+| | Saturn | Giraffe |
```
Comment (Krzysztof-Cieslak) [inline docs/faq-server.md]: I'm not sure if this table is needed at all - image and above descriptions explains it well enough

File: docs/faq-build.md [hunk 2]
```diff
@@ -5,9 +5,7 @@ The SAFE template is geared up towards a streamlined development process. Firstl
 
 The development process uses *two web servers* running in tandem:
 
-* The "back-end" .NET web server which hosts your API endpoints and other server-generated content.
-    * For **[Saturn](component-saturn)** or **Giraffe**, this uses Kestrel on ASP .NET out of the box (although you can opt for [other options]()),
-    * **Suave** uses its own built-in HTTP server.
+* The "back-end" .NET web server which hosts your API endpoints and other server-generated content. This uses Kestrel on ASP .NET out of the box (although you can opt for [other options]()),
```
Comment (Krzysztof-Cieslak) [inline docs/faq-build.md]: Missing link after [other options]

File: docs/faq-server.md [hunk 3]
```diff
@@ -3,28 +3,24 @@
...
 
-In addition, whilst the Giraffe and the Suave programming are extremely composable and fully functional-first, Saturn allows you to take advantage of abstractions such as `application { }`, `scope { }` and `controller { }` which are not only extremely simple to use but will also be familiar to developers from other (non-.NET) web programming models. And since Saturn's abstractions are all *optional*, you can always fall back directly to the lower-level Giraffe model if required.
+In addition, whilst the Giraffe programming is extremely composable and fully functional-first, Saturn allows you to take advantage of abstractions such as `application { }`, `scope { }` and `controller { }` which are not only extremely simple to use but will also be familiar to developers from other (non-.NET) web programming models. And since Saturn's abstractions are all *optional*, you can always fall back directly to the lower-level Giraffe model if required.
```
Comment (Krzysztof-Cieslak) [inline docs/faq-server.md]: We could also mention that Saturn abstractions are actually transformed to standard Giraffe's HttpHandler which means they don't provide too much overhead and they can be easily composed with other Giraffe code.

File: docs/faq-server.md (hunk 1 above)
Comment (isaacabraham) [reply docs/faq-server.md]: Removed

File: docs/faq-build.md (hunk 2 above)
Comment (isaacabraham) [reply docs/faq-build.md]: Fixed.

File: docs/faq-server.md (hunk 3 above)
Comment (isaacabraham) [reply docs/faq-server.md]: That's mentioned elsewhere - perhaps we can rationalise that another time.

**Pull request merged**